frontera/settings/__init__.py

```diff
@@ -60,3 +56,30 @@ def set(self, key, value):
     def set_from_dict(self, attributes):
         for name, value in attributes.items():
             self.set(name, value)
+
+
+class DefaultSettings(FronteraSettings):
+    def __init__(self):
+        super(DefaultSettings, self).__init__(default_settings)
+
+
+class Settings(FronteraSettings):
```

Comment:
I changed __getattr__ to rely on the get method so it doesn't have to be rewritten on subclasses.